setup.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from setuptools import setup
+
+setup(
+    name="mailroom",
+    description="A Python implementation to help the mailroom \
+    create reports or send thank you notes.",
```

Comment:
better to let this be longer than 80 chars.  Splitting it like this is going to leave funny spacing in the result